index.md

```diff
@@ -26,16 +29,16 @@ Jump to:
   {%- endif %}
 {% endfor %}
 
-{%- comment %}
-Limit advertised development releases to those published in the last year
-{%- endcomment %}
+## Development releases
+
+Development releases older than one year aren't listed below,
+but can still be installed with `helm`.
```

Comment:
moved this to the 'development releases' heading instead of repeating it on each.